### PR TITLE
Improve doc and fix legacy api

### PR DESCRIPTION
**Pull request by M0NsTeRRR**

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -175,7 +177,7 @@ func (c Client) GetUsers(usersList []string) ([]UserData, error) {
 	return s.Data, nil
 }
 
-// GetStreams will get a list of live Streams
+// GetFollows will get the number of followers for a user
```
Comment (egguy) [inline client.go:252]: Nice catch ;)

File: client.go [hunk 2]
```diff
@@ -17,15 +17,17 @@ import (
 type StreamData struct {
 	ID           string    `json:"id"`
 	UserID       string    `json:"user_id"`
+	UserLogin    string    `json:"user_login"`
 	UserName     string    `json:"user_name"`
 	GameID       string    `json:"game_id"`
-	CommunityIds []string  `json:"community_ids"`
```
Comment (egguy) [inline client.go:22]: Disappeared ?

File: main.go [hunk 3]
```diff
@@ -109,8 +109,17 @@ func scrapeStreams(twitch *Client) {
 		for id := range channelsID {
 			streamsID = append(streamsID, id)
 		}
-
 		var streamScraped = 0
+
+		// if access token expire in 10 days renew it
+		if time.Now().After(twitch.Token.ExpiresDate.AddDate(0, 0, -10)) {
```
Comment (egguy) [inline main.go]: Why not using a percent remaining like 50% ? If twitch reduce to 24H the code will crash

**Review by egguy — CHANGES_REQUESTED** [review]

File: client.go (hunk 2 above)
Comment (M0NsTeRRR) [reply client.go:22]: Yes
https://dev.twitch.tv/docs/api/reference#get-streams

**Review by egguy — APPROVED** [review]

**Pull request merged**